modules/cluster-logging-about.adoc

```diff
@@ -24,7 +24,7 @@ ifdef::openshift-dedicated[]
 {product-title} administrators can deploy the Red Hat OpenShift Logging Operator and the
 OpenShift Elasticsearch Operator by using the {product-title} web console and can configure logging in the
 `openshift-logging` namespace. Configuring logging will deploy Elasticsearch,
-Fluentd, and Kibana in the `openshift-logging` namespace. The operators are
+collector, and Kibana in the `openshift-logging` namespace. The operators are
```

Comment:
a collector... but seems like there is additional rewrite needed to include loki